<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupportParameter</class>
 <widget class="QWidget" name="SupportParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>516</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>200</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="toolTip">
     <string>底盘设置</string>
    </property>
    <attribute name="title">
     <string>Base</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>形状</string>
       </property>
       <property name="text">
        <string>Shape:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Support_Base_AttachType_comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>半径</string>
       </property>
       <property name="text">
        <string>Radius(mm)：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="Support_Base_Radius_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>30.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.180000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>高度</string>
       </property>
       <property name="text">
        <string>Height(mm)：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="Support_Base_Length_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="top_support">
    <property name="toolTip">
     <string>支撑设置</string>
    </property>
    <attribute name="title">
     <string>Top</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>形状</string>
       </property>
       <property name="text">
        <string>Shape：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Support_Top_AttachType_comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>形状</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>高度</string>
       </property>
       <property name="text">
        <string>Height(mm)：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="Support_Top_Length_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>半径</string>
       </property>
       <property name="text">
        <string>Radius(mm):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="Support_Top_Radius_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>渗透率</string>
       </property>
       <property name="text">
        <string>Penetration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="Support_Top_Penetration_horizontalSlider">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="Support_Top_Penetration_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>角度</string>
       </property>
       <property name="text">
        <string>Angle：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="Support_Top_AngleFactor_horizontalSlider">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="Support_Top_AngleFactor_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="mid_support">
    <property name="toolTip">
     <string>支撑设置</string>
    </property>
    <attribute name="title">
     <string>mid</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>形状</string>
       </property>
       <property name="text">
        <string>Shape：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Support_Mid_AttachType_comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>形状</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>半径</string>
       </property>
       <property name="text">
        <string>Radius(mm)：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="Support_Mid_Radius_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="bottom_support">
    <property name="toolTip">
     <string>支撑设置</string>
    </property>
    <attribute name="title">
     <string>bottom</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>形状</string>
       </property>
       <property name="text">
        <string>Shape：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Support_Bottom_AttachType_comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>高度</string>
       </property>
       <property name="text">
        <string>Height(mm)：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="Support_Bottom_Length_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>半径</string>
       </property>
       <property name="text">
        <string>Radius(mm):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="Support_Bottom_Radius_doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(248, 248,248);</string>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>渗透率</string>
       </property>
       <property name="text">
        <string>Penetration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="Support_Bottom_Penetration_horizontalSlider">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="Support_Bottom_Penetration_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>角度</string>
       </property>
       <property name="text">
        <string>Angle：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="Support_Bottom_AngleFactor_horizontalSlider">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="Support_Bottom_AngleFactor_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonRestoreDefaults">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>467</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTip">
    <string>设置默认参数</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248,248);</string>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>467</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolTip">
    <string>保存设置</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248,248);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>322</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxBase">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>底盘</string>
      </property>
      <property name="text">
       <string>Base</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QSlider" name="Support_Base_Coverage_horizontalSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>193</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>底盘半径</string>
      </property>
      <property name="maximum">
       <number>199</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButtonAdd">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>添加</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButtonDelete">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>删除</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonCleanAll">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>清空所有支撑</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 248,248);</string>
      </property>
      <property name="text">
       <string>Clean All</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>自动支撑密度</string>
      </property>
      <property name="text">
       <string>  Density:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxDensity">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 248,248);</string>
      </property>
      <property name="singleStep">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonAutoSupport">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>自动添加支撑</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 248,248);</string>
      </property>
      <property name="text">
       <string>Auto Support</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QToolButton" name="toolButtonAdvanced">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 248,248);</string>
      </property>
      <property name="text">
       <string>         Advanced Options</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="arrowType">
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButtonRestoreDefaults</zorder>
  <zorder>pushButtonSave</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxBase</sender>
   <signal>toggled(bool)</signal>
   <receiver>Support_Base_Coverage_horizontalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
